t/12_sles4sap_publicccloud.t

```diff
@@ -43,4 +43,50 @@ subtest "Run 'setup_sbd_delay_publiccloud' with different values" => sub {
 
 };
 
+subtest '[is_hana_database_offline]' => sub {
+    my $self = sles4sap_publiccloud->new();
+    my $sles4sap_publiccloud = Test::MockModule->new('sles4sap_publiccloud', no_auto => 1);
+    $sles4sap_publiccloud->redefine(get_hana_database_info => sub { return 0; });
+    $sles4sap_publiccloud->redefine(record_info => sub { return; });
+    set_var('SAP_SIDADM', 'SAP_SIDADMTEST');
+    set_var('INSTANCE_ID', 'INSTANCE_IDTEST');
+
+    my $res = $self->is_hana_database_offline();
+    is $res, 0, "Hana database is offline";
```

Comment:
Just after the `is_hana_database_offline` and just before to `assert`, please add
```
    set_var('SAP_SIDADM', undef);
    set_var('INSTANCE_ID', undef);
```
otherwise other tests will get them too

Reply:
refactored